master/internal/resourcemanagers/kubernetes/spec.go

```diff
@@ -315,6 +315,13 @@ func (p *pod) configurePodSpec(
 			continue
 		}
 
+		for _, e := range container.Env {
```

Comment:
pro/con putting this in all containers vs just the determined container?

Reply:
I thought about this and I didn't think there was a use case for setting env variables for the init container or fluent container. But allowing it would make it feel more true to the k8s experience as opposed to just borking it.

Reply:
yeh, probably never necessary, but idk.. if someone needs to drop aws creds into fluent for the connection to a elasticsearch.. lol idk, we'll get there when we get there

Reply:
probably a better way to do that anyway

Reply:
I'll investigate and if I can support more containers easily, I'll include it